src/styles/overrides/_se-form.scss

```diff
@@ -0,0 +1,92 @@
+.seform {
+  form {
+    .seText label {
+      @extend .dg_label;
+    }
+
+    fieldset {
+      @extend .dg_fieldset;
+
+      legend {
+        @extend .dg_legend;
+      }
+
+      .seRadioLabel {
+        @extend .dg_label;
+        @extend .dg_radio-label;
+      }
+
+      .seCheckboxLabel {
+        @extend .dg_label;
+        @extend .dg_checkbox-label;
+      }
+
+      .seRadio {
+        @extend .dg_radio-input;
+      }
+
+      .seCheckbox {
+        @extend .dg_checkbox-input;
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
```

Comment:
Recommend updating to `$base-spacing-unit`.

Reply:
@theilman7 yes, it was a result of not merging, this file is no longer available.